engine/modules/opentelemetry/doc/opentelemetry.md

```diff
@@ -207,3 +207,97 @@ An example of configuration:
         }
     }
   ```
+
+### centreon monitoring agent
+Event if all protobuf objects are opentelemetry objects, grpc communication is made in streaming mode. It is more efficient, it allows reverse connection (engine can connect to an agent running in a DMZ) and 
+Engine can send configuration on each config update.
+You can find all grpc definitions are agent/proto/agent.proto.
+Every time engine configuration is updated, we calculate configuration for each connected agent and send it on the wire if we find a difference with the old configuration. That's why each connection has a ```agent::MessageToAgent _last_config``` attribute.
+So, the opentelemetry engine server supports two services, opentelemetry service and agent streaming service.
+OpenTelemetry data is different from telegraf one:
+* host service attributes are stored in resource_metrics.resource.attributes
+* performance data (min, max, critical lt, warning gt...) is stored in exemplar, service status is stored in status metric
+  
+Example:
```

Comment:
Could you also give the service output just as given by the plugin, so it is easier to compare.

Reply:
done